cumulusci/core/config/OrgConfig.py

```diff
@@ -120,16 +116,16 @@ def lightning_base_url(self):
         else:
             return self.instance_url.split(".")[0] + ".lightning.force.com"
 
+    # to simplify mocking for tests
+    _Salesforce = Salesforce
```

Comment:
Where in the tests is _Salesforce referenced?

Reply:
This was vestigial from an earlier attempt. Fixed.